BC_loadouts/factions/russia.hpp

```diff
@@ -22,79 +22,166 @@ class russia {
      class Type {
           class Soldier_F {
                gps = "";
-               addItemsToUniform[] = {"ACE_fieldDressing","ACE_MapTools","ACE_Flashlight_KSF1",LIST_2("rhs_mag_9x19_17")};
-               addItemsToVest[] = {"SmokeShell",LIST_4("rhs_30Rnd_545x39_7N10_AK"),LIST_2("rhs_mag_rgd5"),LIST_2("rhs_mag_rdg2_white")};
-               addItemsToBackpack[] = {LIST_6("ACE_fieldDressing"),LIST_2("ACE_epinephrine"),LIST_2("ACE_morphine")};
+               addItemsToUniform[] = {
+                  "ACE_fieldDressing",
+                  "ACE_MapTools",
+                  "ACE_Flashlight_KSF1",
+                  LIST_2("rhs_mag_9x19_17")};
+                   addItemsToVest[] = {"SmokeShell",
+                  LIST_4("rhs_30Rnd_545x39_7N10_AK"),
+                  LIST_2("rhs_mag_rgd5"),
+                  LIST_2("rhs_mag_rdg2_white")};
+                   addItemsToBackpack[] = {LIST_6("ACE_fieldDressing"),
+                  LIST_2("ACE_epinephrine"),
+                  LIST_2("ACE_morphine")
+            };
           };
           class Engineer_F: Soldier_F {
-               addItemsToVest[] = {"SmokeShell",LIST_7("rhs_30Rnd_545x39_7N10_AK"),LIST_2("rhs_mag_rgd5"),LIST_2("rhs_mag_rdg2_white")};
-               backpack = "rhssaf_kitbag_smb";
-               addItemsToBackpack[] = {LIST_6("ACE_fieldDressing"),LIST_2("ACE_epinephrine"),LIST_2("ACE_morphine"),"grad_axe","ACE_Fortify","ACE_EntrenchingTool","ACE_DefusalKit"};
+               addItemsToVest[] = {
+                  "SmokeShell",
+                  LIST_7("rhs_30Rnd_545x39_7N10_AK"),
+                  LIST_2("rhs_mag_rgd5"),
+                  LIST_2("rhs_mag_rdg2_white")};
+                   backpack = "rhssaf_kitbag_smb";
+                   addItemsToBackpack[] = {LIST_6("ACE_fieldDressing"),
```

Comment:
die Zpeilen sind nicht mehr zu lang, aber dafür ist jetzt die Einrückung total b0rked in `Engineer_F` :(

Reply:
Sry not going to fix indentations.